BUILDING.md

```diff
@@ -121,6 +121,8 @@ Changes to these files should become a single commit for a PR which relies on ve
 
 Please refer to [RUNC.md](/docs/RUNC.md) for the currently supported version of `runc` that is used by containerd.
 
+> *Note*: On macOS, building containerd is possible but it will not contain a container runtime.
```

Comment:
```suggestion
> *Note*: On macOS, building containerd is possible but running containerd natively is not yet supported on macOS.
```

Reply:
consider mentioning this issue up on the runc section as well .. cheers!

Reply:
I agree with Mike's suggestion on how to word this, and that might just be due to the overloaded term container runtime lol. It's odd to state building containerd (which our About section on github is "An open and reliable container runtime") will not contain a container runtime.

Reply:
@mxpv I'm curious on how you'd view wording this as well. It definitely "runs"

Reply:
Yeah, I withheld comment worried I might appear as nitpicking, but now that @dcantah opened the floodgates :joy: ..we're kind of in a weird spot with macOS.. we could list a number of PRs that have continued to add viability to using containerd on macOS, especially for those who don't mind experimenting or picking up additional components outside of the containerd repo. Although this phrase ending with "not yet supported" might be a reasonable thing to say. It's hard to say how clear we should be here, but there is a difference between "it won't run" versus "we don't support it yet"

Reply:
Yea it's hard to state support when there's no shim to point folks to. The daemon.. does daemon things and can pull images with the native snapshotter. Like you said there's a bit of a difference between "won't run", it definitely does that, but I'd be fine saying we don't really support it 🤷. As a fun side affair I was trying to get kata to work on darwin, so maybe if that ever gets finished we could change this?

Reply:
Probably worth to distinguish containerd daemon here (which works just fine) and shim runtime for MacOS (which we don't have).

Reply:
> You can build and run the containerd daemon on macOS, but there is no shim runtime currently offered by containerd maintainers that would let you run containers natively on this OS.

?